README.md

```diff
@@ -127,6 +104,7 @@ Additionally if you are not using the development container:
 
 ### Tests
 
-The project contains unit and regression testing using pytest that can be run by:
+The project contains unit and regression testing using pytest that can be run
+by:
 
     pytest tests
```

Comment:
Maybe it could be mentioned here that one needs to have the appropriate kernel sources downloaded and configured to run the tests and how to do that.